src/components/CommentSection.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useState } from 'react';
+import { RefObject } from 'react';
+import { useRef } from 'react';
+import { useMutation } from 'react-apollo';
+import { Alert, ScrollView, StyleSheet, TextInput, View } from 'react-native';
+import { normalize } from 'react-native-elements';
+
+import { colors, device, texts } from '../config';
+import { momentFormat } from '../helpers';
+import { COMMENT_ON_SURVEY } from '../queries/survey';
+import { Survey } from '../types';
+
+import { Button } from './Button';
+import { RegularText } from './Text';
+import { Wrapper } from './Wrapper';
+
+type Props = {
+  archived?: boolean;
+  comments: Survey['surveyComments'];
+  scrollViewRef: RefObject<ScrollView>;
+  surveyId: string;
+};
+
+const MAX_COMMENT_LENGTH = 5000;
+
+export const CommentSection = ({ archived, comments, scrollViewRef, surveyId }: Props) => {
+  const refForPosition = useRef<View>(null);
+  const [sendComment] = useMutation(COMMENT_ON_SURVEY);
+  const [newComment, setNewComment] = useState('');
+
+  const submitComment = useCallback(() => {
+    setNewComment('');
+    sendComment({ variables: { surveyId, message: newComment } });
+    Alert.alert(
+      texts.survey.commentSubmissionAlert.de + '\n\n' + texts.survey.commentSubmissionAlert.pl
+    );
```

Comment:
not optimal not seeing the whole text (_android 7.0 device_)

![Screenshot_20210707-151512](https://user-images.githubusercontent.com/1942953/124765556-5e677a80-df36-11eb-92d4-20761693dfb7.png)

Reply:
added a shorter title
![Screenshot 2021-07-07 at 15 35 59](https://user-images.githubusercontent.com/59824597/124770850-f5cecc80-df3a-11eb-9a0c-1ad4dcca1672.png)
![Screenshot_20210707-153743](https://user-images.githubusercontent.com/59824597/124770895-fff0cb00-df3a-11eb-8258-468cda2f0e7c.png)

Reply:
yes, agree

![Screenshot_20210707-155810.png](https://user-images.githubusercontent.com/1942953/124772489-5dd1e280-df3c-11eb-879f-8353803e4b88.png)